src/widgets/forms/autocomplete/autocompleteField.ts

```diff
@@ -102,13 +103,13 @@ export function autocompleteField (
   const kb = store
   const formDoc = form.doc ? form.doc() : null // @@ if blank no way to know
 
-  const box = dom.createElement('tr')
+  const box = dom.createElement('div')
```

Comment:
**praise**: Good to move away from table based layout

Reply:
👍